### PR TITLE
quiz to moodle UI

### PR DESCRIPTION
**Pull request by JoneiceB**
This is my send quiz to moodle UI

### CODE REVIEW

File: teamA/lib/Views/send_quiz_to_moodle.dart [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+  primaryContainer: Color(0xFFEDE6FF), // Light lavender
+  onPrimaryContainer: Color(0xFF2D004A), // Dark purple
+  secondary: Color(0xFF6A5A99), // Using similar purple as secondary
+  onSecondary: Colors.white, // Black text on background
+  surface: Color(0xFFFFFFFF), // White surface color
+  onSurface: Colors.black, // Black text on surface
+  error: Colors.red, // Error color red
+  onError: Colors.white, // White on error
+  brightness: Brightness.light, // Light mode
+);
```
Comment (DavidWorthy) [inline teamA/lib/Views/send_quiz_to_moodle.dart]: Color scheme can be dropped.

**Review by DavidWorthy — CHANGES_REQUESTED** [review]
Screen is inaccessible from the dev page, so I can't see what's there. Please adjust that.

File: teamA/lib/Views/send_quiz_to_moodle.dart (hunk 1 above)
Comment (JoneiceB) [reply teamA/lib/Views/send_quiz_to_moodle.dart]: Done

**Comment by JoneiceB** [comment]
Screen should now be accessible from dev page.

File: teamA/lib/main.dart [hunk 2]
```diff
@@ -45,6 +46,7 @@ class MyApp extends StatelessWidget {
         // '/create': (context) => const CreatePage(),
         '/dashboard': (context) => TeacherDashboard(),
         '/send_essay_to_moodle': (context) => EssayAssignmentSettings(),
+        '/send_quiz_to_moodle': (contet) => QuizMoodle(),
```
Comment (DavidWorthy) [inline teamA/lib/main.dart]: Misspelled context.

File: teamA/lib/Views/send_quiz_to_moodle.dart [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+                  textAlign: TextAlign.center,
+                ),
+              ),
+            ),
+            SizedBox(
+              height: 30,
+            ),
+
+            sectionTitle(title: 'Quiz Information'),
+            TextField(
```
Comment (DavidWorthy) [inline teamA/lib/Views/send_quiz_to_moodle.dart]: Course name should really be a dropdown with its contents taken from the course list, but that second part can be handled as part of the BL.

**Comment by JoneiceB** [comment]
Updated the 'send quiz to moodle' UI to have the dropdown button for course name. Also updated the 'course list' UI as well.

**Review by DavidWorthy — APPROVED** [review]
All good. Part of the reason why I didn't review the other branch after the changes is that there were some... oddities in some of additional files. Now that the course list is also implemented here, I can close both branches.

**Pull request merged**